tests/testthat/test-parse-expr-debug.R

```diff
@@ -0,0 +1,55 @@
+test_that("can parse trivial print statement", {
+  expect_equal(
+    parse_expr(quote(print("{a}")), NULL, NULL),
+    list(special = "print",
+         rhs = list(type = "print"),
+         string = "{a}",
+         inputs = list(list(expr = quote(a), format = NULL)),
+         depends = list(functions = character(), variables = "a"),
+         when = NULL))
+})
+
+
+test_that("debug string requires at least one variable", {
+  expect_error(
+    parse_expr(quote(print("a string")), NULL, NULL),
+    "Invalid 'print()' expression does not reference any values",
+    fixed = TRUE)
+})
```

Comment:
the concern was mostly that most of the time this would be a mistake, and was to help them fix it.  But don't feel strongly about this at all!

We've had a longstanding ticket for assertions that might be nice for the panic option